### ISSUE TITLE
Homebrew tap asks for github credentials

### ISSUE THREAD

**Issue by julianxhokaxhiu** [riboseinc/homebrew-jenkins-slave#12]
Hi,

I'm trying to use this cask to install the Jenkins worker node agent, but upon launching this command, it asks for github credentials.
```sh
$ brew tap iteratec/jenkins-slave
Cloning into '/usr/local/Homebrew/Library/Taps/iteratec/homebrew-jenkins-slave'...
Username for 'https://github.com':
```
Is this a left over from a previous fork?

Thank you in advance,
Julian

**Comment by Nadav-Ruskin** [comment]
I ran into the same issue. Out of instinct my solution was to use:
```
brew tap riboseinc/jenkins-slave
```
Seems to work fine.

**Comment by ronaldtse** [comment]
@julianxhokaxhiu @Nadav-Ruskin thank you for letting us know! Indeed that was a mistake committed from someone's contributed PR. It's now fixed in 6c554a2 .
